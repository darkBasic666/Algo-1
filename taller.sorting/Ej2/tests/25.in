25 0
[(Ruiz de Montoya;10),(La Paternal;35),(Rosario de La Frontera;14),(Río Segundo;44),(Los Toldos;17),(Villa Guillermina;1),(Gral. Cabrera;15),(El Talar;47),(La Criolla;29),(Tapso;29),(Tres Lagunas;3),(Galisteo;21),(Aldea Sta. Rosa;18),(Nicanor E. Molinas;38),(Arrayanal;8),(Atahona;4),(Huinca Renanco;36),(Lavaisse;8),(Gdor. Echagüe;47),(Candioti;17),(Capioví;18),(Los Chanaritos (R.S);37),(Alpa Corral;13),(Arocena;14),(Col. Alberdi;46)]
