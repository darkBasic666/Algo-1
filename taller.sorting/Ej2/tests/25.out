25 0
[(Villa Guillermina;1),(Tres Lagunas;3),(Atahona;4),(Arrayanal;8),(Lavaisse;8),(Ruiz de Montoya;10),(Alpa Corral;13),(Arocena;14),(Rosario de La Frontera;14),(Gral. Cabrera;15),(Candioti;17),(Los Toldos;17),(Aldea Sta. Rosa;18),(Capioví;18),(Galisteo;21),(La Criolla;29),(Tapso;29),(La Paternal;35),(Huinca Renanco;36),(Los Chanaritos (R.S);37),(Nicanor E. Molinas;38),(Río Segundo;44),(Col. Alberdi;46),(El Talar;47),(Gdor. Echagüe;47)]
